fiftyone/migrations/revisions/v0_6_2.py

```diff
@@ -0,0 +1,21 @@
+"""
+FiftyOne v0.6.2 revisions
```

Comment:
Are you expecting this to be in 0.6.1 or 0.6.2?

Reply:
0.6.2